### PR TITLE
Hyundai KONA EV 2021: update tuning

### PR DESCRIPTION
**Pull request by paul-lee-dev**
For smoother maintenance center lane in Kona EV.
Change pid -> indi

### CODE REVIEW

**Comment by sshane** [comment]
Should we hold off on changing any tunes to INDI in light of the new torque controller?

**Comment by speedking456** [comment]
> Should we hold off on changing any tunes to INDI in light of the new torque controller soon to expand to more cars? @HaraldSchafer

I've been playing around with the torque controller on my Kia Seltos, feels a lot better and was straightforward to measure the values and tune it accordingly. Any new tunes should be using that controller imo.

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class CAR:
   IONIQ_PHEV_2019 = "HYUNDAI IONIQ PLUG-IN HYBRID 2019"
   IONIQ_PHEV = "HYUNDAI IONIQ PHEV 2020"
   KONA = "HYUNDAI KONA 2020"
-  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
+  KONA_EV = "HYUNDAI KONA ELECTRIC 2018-2021"
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:45]:
```suggestion
  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
```

**Comment by sshane** [comment]
@cebu13 a guide will probably be written shortly for how to gather data for the torque controller, but in the mean time, experiment with that controller instead of INDI or PID. A table can be found here: https://discord.com/channels/469524606043160576/574796986822295569/966407640425058335

Try something out close to your car

**Comment by Azenzic** [comment]
> https://discord.com/channels/469524606043160576/574796986822295569/966407640425058335
doesn't seem to be a Channel in the discord that's open to the public.

**Comment by cebu13** [comment]
@sshane Looking forward to the guide. I will look into it. Thanks.

**Pull request closed without merging**